docs/UserGuide.adoc

```diff
@@ -737,6 +761,14 @@ Opens an email draft in the browser with recipients as all persons tagged with '
 
 *Q*: How do I transfer my data to another Computer? +
 *A*: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous Address Book folder.
+
+*Q*: Why is it that an error shows up when I clicked the interface of the calendar twice, as seen in Figure ? below?
+
```

Comment:
`Figure ?` is a placeholder right?

Reply:
Yea. Once all the pictures are in they will be replaced. It was suggested to refer to the figure number directly.